### ISSUE TITLE
[S]  Values where multiple options can be selected in dropdown on save/import do not show both fields as selected on edit for Staff Relationship

### ISSUE THREAD

**Issue by crisr15** [scientist-softserv/britishlibrary#149]
I think this has partly been fixed since originally logged - but the issue remains with multiple values in dropdown lists, namely creator/contrib/editor staff relationship dropdown list, and the Institution list (i.e. BL, BM, Kew, NMS, MOLA).

Scenario 1: 
Using Bulkrax to import. If we have TRUE in both 'staff member' and 'research associate' columns for a creator/contrib/editor - when we go to edit the work in the app, we should be able to see both values highlighted in the relationship dropdown list for each name. 

To recreate the issue:
Use [staffrelshiptest20220929_03_both.csv](/uploads/a4d779e9d8b85fea4c535c443927fd29/staffrelshiptest20220929_03_both.csv) to create a work with a creator with both research associate and staff member status. 

Here's my importer for that work: https://bl-demo.iro.bl.uk/importers/83?locale=en 
The work is here: https://bl-demo.iro.bl.uk/concern/books/73a1982a-b805-4df6-95bf-2b48bf12618d?locale=en 

When this ticket was originally logged, the values didn't show in the app but they did show in the export. However now on [export](https://bl-demo.iro.bl.uk/exporters/26?locale=en), neither staff relationship (staff/RA) is showing in the [export_26_66.zip](/uploads/028ad30707dca3b27aacf22a7fe97aef/export_26_66.zip).

Something to bear in mind in the solution:
We will be adding a third category of collaborative doctoral placement student so the rule for it should be showing multiple, not showing two.

Scenario 2:
This multiple dropdown list issue also occurs with Institution metadata field if you have more than one institution selected, which is rare but could happen if we collaborate on projects (i.e. BL, MB, Kew etc). 

This [importer](https://bl-demo.iro.bl.uk/importers/85?locale=en) has the same works, each has two institutions. They import fine because you can see them when you see the public view of the work. But when you go into the edit work screen of the record, it only shows one of them. However on export, both institutions can be seen.

Work: https://bl-demo.iro.bl.uk/concern/articles/610ecb39-d0ba-4b7e-843d-882d3adb01ad?locale=en 
Screenshot of public page: ![multiinstitutions](/uploads/3bae5a25133fe42cb377c050272e46c0/multiinstitutions.JPG)
And screenshot of it in the edit view only highlighting the BL value: ![multiinstitutions_editscreen](/uploads/41ac07c3cf2ff3bad1513038c1d27364/multiinstitutions_editscreen.JPG)

To recreate the issue:
Import file: [testinginstitutioninUIandexport_20220930.csv](/uploads/72c0c2ab3a4259dd25f7eaf06172679f/testinginstitutioninUIandexport_20220930.csv)

Importer I used: https://bl-demo.iro.bl.uk/importers/85?locale=en
Exporter I used: https://bl-demo.iro.bl.uk/exporters/28?locale=en
Works it produced: 
1. https://bl-demo.iro.bl.uk/concern/reports/267e5010-e92d-4ca5-8a64-bf0ce33b7505?locale=en
2. https://bl-demo.iro.bl.uk/concern/books/06cbc48f-91cf-4da7-a492-7631dee7844c?locale=en
3. https://bl-demo.iro.bl.uk/concern/articles/610ecb39-d0ba-4b7e-843d-882d3adb01ad?locale=en 

(all three works have two institutions selected)

In summary: for institution issue, the two values are showing on public view, import and export, just not in edit screen.

For relationship issue, the two values are showing on import but not on edit screen or export (they do not show in public view by design).
